### DISCUSSION TITLE
Brother MFC-J4620DW works over WSD

### DISCUSSION THREAD

**Discussion by hijackeel** [alexpevzner/sane-airscan#241] [General]
"If you have success with a scanner not included into this list, please let me know."

Reporting that Brother MFC-J4620DW works over WSD. :)

`scanimage -L`
`device airscan:w1:Brother MFC-J4620DW' is a WSD Brother MFC-J4620DW ip=xxx.xxx.xxx.xxx`

**Comment by alexpevzner** [comment]
Thanks for reporting that! README.md updated.
